packages/web/src/scss/components/Accordion/index.html

```diff
@@ -0,0 +1,450 @@
+{{#> layout/plain }}
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default accordion (Always open)</h2>
```

Comment:
```suggestion
  <h2 class="docs-Heading">Default accordion (Stay open)</h2>
```

I think that `stay open` better describes this functionality.

Reply:
But I think I should by ok with this. Bootstrap is also using it (https://getbootstrap.com/docs/5.2/components/accordion/#always-open) 🤷